db/data/regions/_default/en.yml

```diff
@@ -8,117 +8,171 @@ en:
-              %{zip}.
-            unknown_for_address: Building number may be incorrect.
+            contains_restricted_characters_informative: Building number can't contain restricted characters
+            contains_html_tags_instructional: Building number can't contain HTML
+            contains_html_tags_informative: Building number can't contain HTML
+            contains_url_instructional: Building number can't contain a URL
+            contains_url_informative: Building number can't contain a URL
+            building_number_invalid_instructional: Building number couldn't be located for %{street},
+              %{zip}
+            building_number_invalid_informative: Building number couldn't be located for %{street},
+              %{zip}
+            unknown_for_address_instructional: Enter valid building number
+            unknown_for_address_informative: Building number invalid
           warnings:
             contains_too_many_words: Building number is recommended to have less than %{word_count} words
         address2:
           label:
             default: Apartment, suite, etc.
             optional: Apartment, suite, etc. (optional)
           errors:
-            blank: Enter an apartment, suite, etc.
-            too_long: The second address line is too long (maximum is 255 characters)
-            contains_emojis: Address2 cannot contain emojis
-            contains_mathematical_symbols: Address2 cannot contain mathematical symbols
-            contains_restricted_characters: Address2 can only contain letters, numbers,
+            blank_instructional: Enter an apartment, suite, etc.
+            blank_informative: Apartment or suite is missing
+            too_long_instructional: The second address line is too long (maximum is 255 characters)
+            too_long_informational: The second address line is too long
+            contains_emojis: Address line 2 can't contain emojis
+            contains_mathematical_symbols: Address line 2 can't contain mathematical symbols
+            contains_restricted_characters: Address line 2 can only contain letters, numbers,
               local characters, and special characters
-            contains_too_many_words: Address2 cannot have more than %{word_count}
+            contains_too_many_words: Address line 2 can't have more than %{word_count}
               words
-            contains_html_tags: Address2 cannot contain HTML tags.
-            contains_url: Address2 cannot contain URLs.
-            missing_building_number: Add a building number if you have one.
-            street_unknown_for_zip: Enter a valid street name for %{zip}.
-            building_number_invalid: Building number couldn't be located for %{street},
-              %{zip}.
-            unknown_for_address: Address line 2 may be incorrect.
+            contains_html_tags: Address line 2 can't contain HTML
+            contains_url: Address line 2 can't contain a URL
```

Comment:
So difficult to know exactly what this will look like, but we should align on one term at least.

Let's make it `Address line 2` for everything? (cc @ksvihar)